.gitignore

```diff
@@ -0,0 +1,2 @@
+.env 
+/node_modules
```

Comment:
it'd be good to add `.DS_Store` to this gitignore. that file just gets generated by the mac OS when you open a directory in Finder, and never needs to be committed